notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb

```diff
@@ -85,9 +97,9 @@
         "\n",
         "The steps performed include:\n",
         "\n",
-        "- Upload a pre-trained model as a `Model` resource.\n",
-        "- Run a `BatchPredictionJob` on the `Model` resource with ground truth data.\n",
-        "- Generate evaluation `Metrics` artifact about the `Model` resource.\n",
+        "- Upload a pre-trained model as a Model resource.\n",
```

Comment:
- change "pre-trained" to "pretrained"
- change "Model resource" to "model resource"